package.json

```
@@ -32,7 +32,7 @@
   },
   "scripts": {
     "build-app": "cd app && webpack",
-    "build-app-static": "ncp app/src/static/ app/lib/static/ && ncp app/dist/preload.js app/lib/preload.js && ncp app/dist/preload.js.map app/lib/preload.js.map",
+    "build-app-static": "./.github/build-app-static.js",
```

This fails the build on Windows. I think you want
```suggestion
    "build-app-static": "node .github/build-app-static.js",
```

---

Do you think I should leave the shebang (at the top of the file) in that case? Or should I remove it?

---

> Do you think I should leave the shebang (at the top of the file) in that case? Or should I remove it?

@hqurve I'd leave it, it remains valuable.